lib/config/schema.json

```diff
@@ -138,6 +138,9 @@
         },
         "lazyRoutePrefix": {
           "type": "string"
+        },
+        "poll": {
+          "type": [ "boolean", "number" ]
```

Comment:
This ends up causing a warning in SchemaClassFactory._createChildProperty when doing an `ng serve` because the function isn't built to handle a field that can be of more than one type. The example `"type": [ "boolean", "number" ]` is valid JSON Schema per http://jsonschemalint.com/draft5/